recipes/pytest/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+$PYTHON setup.py install --old-and-unmanageable
```

Comment:
Could we please change `--old-and-unmanageable` to ` --single-version-externally-managed --record=record.txt`?